kpatch-build/kpatch-build

```diff
@@ -186,16 +204,6 @@ gcc_version_check() {
 	#   <linux>/arch/powerpc/tools/gcc-check-mprofile-kernel.sh
 	#
 	# Postpone the check to the kernel build instead of duplicating it here.
```

Comment:
Now that we're doing the check for all arches, these ppc64le specific comments can be removed.

Reply:
I wanted to keep as reference that `-mprofile-kernel` is required on ppc64le. But if you think it's not necessary I'll remove it.